PeerioTechnologies/peerio-desktop
# style: fix various Inputs broken by new Input style

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/12985/desktop-upload-preview-dialog-slightly-borked
https://app.clubhouse.io/peerio/story/13000/desktop-room-title-doesn-t-look-good-with-new-inputs
https://app.clubhouse.io/peerio/story/13395/desktop-tab-between-displayname-fields-in-signup
(no story for SecuritySettings 2fa input, I just spotted it)

#### Testing instructions  
Fixed input styling that was broken by redesign from the new Signup flow
* room title editor
* "Share with" input in upload-to-chat dialog
  * search bar in ShareWith dialog
* 2fa input in SecuritySettings

Also in the Signup flow, tabbing between first name and last name inputs should skip the "X" button now

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
